app/redux/Store.js

```diff
@@ -52,6 +53,7 @@ export default initializeStore = () => {
             'history',
             'killSwitch',
             'suggestions',
+            'appState',
         ],
         // note, everything in sets is to be persisted so not blacklisting or transforming them
         transforms: [
```

Comment:
add wasEdited (or whatever you rename it to) to settings filter

Reply:
???

Reply:
Done